packages/UsageAnalysis/files/Test Track/Viewers/Pivot table.md

```diff
@@ -1,9 +1,31 @@
-Check tickets:
+### Pivot table
 
-* [#2606](https://github.com/datagrok-ai/public/issues/2606)
-* [#2198](https://github.com/datagrok-ai/public/issues/2198)
-* [#2497](https://github.com/datagrok-ai/public/issues/2497)
-* [#2535](https://github.com/datagrok-ai/public/issues/2535)
+1. Open the **demog** dataset.
```

Comment:
Link dataset to the test case

Reply:
added   "datasets": ["System:DemoFiles/demog.csv"]